### PR TITLE
solution was added

### PR DESCRIPTION
**Pull request by Maksym845**
- [DEMO LINK](https://Maksym845.github.io/react_list-of-goods-hooks/)

### CODE REVIEW

**Review by Yeliseiev — APPROVED** [review]
Good job!

File: src/App.tsx [hunk 1]
```diff
@@ -15,47 +16,100 @@ export const goodsFromServer = [
...
+    visibleGoods.sort((a, b) => (a.length - b.length));
+  } else if (sortType === SortType.ALPABET) {
+    visibleGoods.sort((a, b) => (a.localeCompare(b)));
+  }
+
+  if (isReversed) {
+    visibleGoods = visibleGoods.reverse();
+  }
+
+  // eslint-disable-next-line no-console
```
Comment (Andr1yK) [inline src/App.tsx:48]: do u need this?

File: src/App.tsx [hunk 2]
```diff
@@ -15,47 +16,100 @@ export const goodsFromServer = [
...
-          <li data-cy="Good">Dumplings</li>
-          <li data-cy="Good">Carrot</li>
-          <li data-cy="Good">Eggs</li>
-          <li data-cy="Good">Ice cream</li>
-          <li data-cy="Good">Apple</li>
-          <li data-cy="Good">...</li>
+          {getReorderedGoods(goodsFromServer, {
+            sortType,
+            isReversed,
+          }).map(good => (<li data-cy="Good" key={good}>{good}</li>))}
```
Comment (Andr1yK) [inline src/App.tsx:112]: move to constant :)

IMHO, in jsx we need only write some render))

**Review by Andr1yK — CHANGES_REQUESTED** [review]

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]
Small fixes and it will be good to go

**Review by sergii-nosachenko — APPROVED** [review]
Good job 🚀🚀🚀

**Pull request closed without merging**